src/lib/utils/SynapseClient.ts

```diff
@@ -827,6 +827,22 @@ export const getEntity: GetEntity = <T>(
   ) as Promise<T>
 }
 
+/**
+ * Get a list of entity headers given by entity ids
+ * http://rest-docs.synapse.org/rest/GET/entity/type.html
+ */
+export const getEntityTypeByIds = <T extends PaginatedResults<EntityHeader>> (
```

Comment:
These are not just the entity types that are being returned.  Could you rename to getEntityHeadersByIds?

Reply:
This is done.